--- third_party/boringssl/src/crypto/bio/socket_helper.c~	2015-07-14 01:16:57.000000000 +0200
+++ third_party/boringssl/src/crypto/bio/socket_helper.c	2015-11-21 10:25:59.264814432 +0100
@@ -12,7 +12,6 @@
  * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
  * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
 
-#define _POSIX_SOURCE
 
 #include <openssl/bio.h>
 #include <openssl/err.h>
@@ -22,6 +22,7 @@
 #include <sys/types.h>
 
 #if !defined(OPENSSL_WINDOWS)
+#include <sys/socket.h>
 #include <netdb.h>
 #include <unistd.h>
 #else
