--- modpagespeed/src/net/instaweb/util/hostname_util.cc~	2013-03-02 01:53:18.000000000 +0200
+++ modpagespeed/src/net/instaweb/util/hostname_util.cc	2013-03-02 11:28:01.761343397 +0200
@@ -19,6 +19,7 @@
 #include "net/instaweb/util/public/hostname_util.h"
 
 #include <limits.h>
+#include <unistd.h>
 // The following break portability.
 
 // Windows doesn't use <unistd.h> nor does it define HOST_NAME_MAX.
