--- modpagespeed/src/third_party/apr/apr.gyp~	2012-06-24 15:31:59.000000000 +0300
+++ modpagespeed/src/third_party/apr/apr.gyp	2012-07-07 14:44:06.094870546 +0300
@@ -19,7 +19,7 @@
     'apr_src_root': '<(apr_root)/src',
     'apr_gen_os_root': '<(apr_root)/gen/arch/<(OS)',
     'apr_gen_arch_root': '<(apr_gen_os_root)/<(target_arch)',
-    'system_include_path_apr%': '/usr/include/apr-1.0',
+    'system_include_path_apr%': '/usr/include/apr',
     'conditions': [
       ['OS!="win"', {
         'apr_os_include': '<(apr_src_root)/include/arch/unix',
--- modpagespeed/src/third_party/aprutil/aprutil.gyp~	2012-06-24 15:31:59.000000000 +0300
+++ modpagespeed/src/third_party/aprutil/aprutil.gyp	2012-07-07 14:48:10.299435236 +0300
@@ -22,7 +22,7 @@
     'aprutil_src_root': '<(aprutil_root)/src',
     'aprutil_gen_os_root': '<(aprutil_root)/gen/arch/<(OS)',
     'aprutil_gen_arch_root': '<(aprutil_gen_os_root)/<(target_arch)',
-    'system_include_path_aprutil%': '/usr/include/apr-1.0',
+    'system_include_path_aprutil%': '/usr/include/apr-util',
   },
   'conditions': [
     ['use_system_apache_dev==0', {
--- modpagespeed/src/third_party/httpd/httpd.gyp~	2012-06-24 15:31:59.000000000 +0300
+++ modpagespeed/src/third_party/httpd/httpd.gyp	2012-07-07 15:15:13.598943424 +0300
@@ -18,7 +18,7 @@
     'apache_src_root': '<(apache_root)/src',
     'apache_gen_os_root': '<(apache_root)/gen/arch/<(OS)',
     'apache_gen_arch_root': '<(apache_gen_os_root)/<(target_arch)',
-    'system_include_path_httpd%': '/usr/include/apache2',
+    'system_include_path_httpd%': '/usr/include/apache',
     'conditions': [
       ['OS!="win"', {
         'apache_os_include': '<(apache_src_root)/os/unix',
--- modpagespeed/src/build/install.gyp~	2012-06-24 15:31:45.000000000 +0300
+++ modpagespeed/src/build/install.gyp	2012-07-07 15:41:33.416812382 +0300
@@ -43,7 +43,6 @@
         ],
         'packaging_files_binaries': [
           '<(PRODUCT_DIR)/libmod_pagespeed.so',
-          '<(PRODUCT_DIR)/libmod_pagespeed_ap24.so',
         ],
         'flock_bash': ['flock', '--', '/tmp/linux_package_lock', 'bash'],
         'deb_build': '<(PRODUCT_DIR)/install/debian/build.sh',
